modules/nf-core/kraken2/build/main.nf

```diff
@@ -1,36 +1,39 @@
 process KRAKEN2_BUILD {
-    tag "$meta.id"
+    tag "${meta.id}"
     label 'process_medium'
     conda "${moduleDir}/environment.yml"
-    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://community-cr-prod.seqera.io/docker/registry/v2/blobs/sha256/29/29ed8f68315625eca61a3de9fcb7b8739fe8da23f5779eda3792b9d276aa3b8f/data' :
-        'community.wave.seqera.io/library/kraken2_coreutils_pigz:45764814c4bb5bf3' }"
+    container "${workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container
+        ? 'https://community-cr-prod.seqera.io/docker/registry/v2/blobs/sha256/0f/0f827dcea51be6b5c32255167caa2dfb65607caecdc8b067abd6b71c267e2e82/data'
+        : 'community.wave.seqera.io/library/kraken2_coreutils_pigz:920ecc6b96e2ba71'}"
 
     input:
-    tuple val(meta), path(db)
+    tuple val(meta), path(library_added_files, stageAs: "database/library/added/")
+    tuple val(meta2), path(seqid2taxid_map, stageAs: "database/seqid2taxid.map")
+    tuple val(meta3), path(taxonomy_files, stageAs: "database/taxonomy/")
```

Comment:
My experience was that you cannot stage entire directories. One has to output and input individual files. With that in mind, shouldn't this be like below?

```suggestion
    tuple val(meta), path(library_added_files, stageAs: "database/library/added/*")
    tuple val(meta2), path(seqid2taxid_map, stageAs: "database/seqid2taxid.map")
    tuple val(meta3), path(taxonomy_files, stageAs: "database/taxonomy/*")
```